### DISCUSSION TITLE
contactform module - Incorrect path to TPL file

### DISCUSSION THREAD

**Discussion by Caleydon** [PrestaShop/PrestaShop#35598] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md).
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add attachments

In the contactform module is a path to contactform.tpl defined in incorrect way.

In the module is path defined as: 
`return $this->display(__FILE__, 'views/templates/widget/contactform.tpl');`

but contactform.tpl file is in the module structure in the path:
`views/templates/hook/`

Vice versa in the theme override is contactform.tpl file present in the path:
` themes/classic/modules/contactform/views/templates/widget/`

It should be unified because when the module will be used with 3rd party theme without overrided contactform.tpl file, the module itself will wont find contactform.tpl file in the module folder.

### Steps to reproduce

1. install contactform module
2. delete /themes/classic/modules/contactform/views/templates/widget/contactform.tpl
3. Contact page will return message "No template found for module contactform"

### PrestaShop version(s) where the bug happened

8.1.4

### If your bug is related to a module, specify its name and its version

contactform v4.4.2

**Comment by Caleydon** [comment]
Why the bug report was moved into Discussion section?

**Comment by matks** [comment]
> [Caleydon](https://github.com/Caleydon) started this conversation in [Ideas](https://github.com/PrestaShop/PrestaShop/discussions/categories/ideas)

I think you clicked the wrong button

**Comment by Caleydon** [comment]
In the body of my post are sections "PrestaShop version(s) where the bug happened", "Steps to reproduce" etc. 
The Discussion doesn't contain these sections. I don't know who and why my Bug report was moved into Discussion.

**Comment by Caleydon** [comment]
An here is a information:

Converted from issue
This discussion was converted from issue #35595 on March 11, 2024 09:30. 

The conversion made [paulnoelcholot](https://github.com/paulnoelcholot)
